controllers/rollingupgrade_controller_test.go

```diff
@@ -2132,6 +2145,14 @@ func TestWaitForTermination(t *testing.T) {
 	g.Expect(unjoined).To(gomega.BeTrue())
 }
 
+func buildManager() (manager.Manager, error) {
+	err := upgrademgrv1alpha1.AddToScheme(scheme.Scheme)
```

Comment:
@eytan-avisror This fixes this error:

```
"failed to update status: no kind is registered for the type v1alpha1.RollingUpgrade in scheme \"pkg/runtime/scheme.go:101\"
```

and after scheme registration there is a new one, because the test framework doesn't setup state properly: 


```
"failed to update status: rollingupgrades.upgrademgr.keikoproj.io \"foo\" not found"
```